scripts/Modules/Saritasa.Cake/Context/CommonContext.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.IO;
+using Cake.Common;
+using Cake.Core;
+using Cake.Frosting;
+
+namespace Saritasa.Cake;
+
+/// <summary>
+/// Common context.
+/// </summary>
+public class CommonContext : FrostingContext
+{
+    /// <summary>
+    /// Path to solution directory.
+    /// </summary>
+    /// <remarks>
+    /// Relative path built from "Saritasa.Cake.csproj" folder.
+    /// DefaultValue - "..\\..\\..\\".
+    /// Was override if invoke "Saritasa.Cake.ps1" with "solutionDir" argument (.\Saritasa.Cake.ps1 --solutionDir ..\\SolutionFolder\\).
```

Comment:
I don't think anyone would provide custom `solutionDir`. It's fine as long as it works correctly from powershell/bash scripts.